Partial-CI-Tests-Structure.md

```diff
@@ -0,0 +1,17 @@
+## Introduction
+
+If your PR fails due to the `check_ci_test_suites_to_run` workflow, please look at the error log and depending on the error, do the following:
+
+* If you see an error about the root files config, then see the [root files config section](#root-files-config) below.
```

Comment:
There's only one bullet point here ... are there other types of errors? Or should they do something else (catch-all) if they run into some error that's not this one?

Reply:
Hm, this was the only error that I felt needed explanation, the others can be explained by the overview of this system and the errors are self explanatory.

Reply:
OK, but a one-bullet-point list is still quite weird. At the very least, you should probably have another bullet point that says something like "If you see any other error, ..."

Reply:
Done.